core/src/Kokkos_Graph.hpp

```diff
@@ -148,10 +148,20 @@ Graph<ExecutionSpace> create_graph(ExecutionSpace ex, Closure&& arg_closure) {
   return rv;
 }
 
+template <class ExecutionSpace = DefaultExecutionSpace>
+std::enable_if_t<Kokkos::is_execution_space_v<ExecutionSpace>,
+                 Graph<ExecutionSpace>>
+create_graph(ExecutionSpace exec = ExecutionSpace{}) {
+  return Kokkos::Impl::GraphAccess::construct_graph(std::move(exec));
```

Comment:
Is it ok to `std::move(exec)` when `exec` is not `ExecutionSpace&&` ?

I find the interface a little bit weird about ownership here, but it looks consistent with the previous code.

Reply:
I guess that `ExecutionSpace` is in this case always deduced as a "bare" type (*i.e.* `std::remove_cvref_t<ExecutionSpace> == ExecutionSpace`).

Reply:
It is fine.  This is a classic "pass by value and move" idiom.